### PR TITLE
Error catalogue

### PR DESCRIPTION
**Pull request by gcappello**

### CODE REVIEW

File: app/views/partials/swagger/error-catalogue.hbs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      <span class="hljs-attr">"message"</span>: <span class="hljs-string">"This a sample message"</span>,
+      <span class="hljs-attr">"extensions"</span>: {
+        <span class="hljs-attr">"code"</span>: <span class="hljs-string">"ERROR_CODE"</span>
+      }
+    }
+  ]
+}
+</code></pre>
+
+            <!-- </div> -->
```
Comment (gcappello) [inline app/views/partials/swagger/error-catalogue.hbs]: Maybe we can remove this line and the empty spaces

File: README.md [hunk 2]
```diff
@@ -74,6 +73,22 @@ domains:
      - name: Invoke Mutation # Mutation 
        description: Markdown enabled description for operation
        query: mutation.mutateSome # Mutation example - invoke mutation
+
+# define your errors byr providing a list
```
Comment (adrianet86) [inline README.md]: Typo

File: app/views/partials/swagger/error-catalogue.hbs (hunk 1 above)
Comment (adrianet86) [reply app/views/partials/swagger/error-catalogue.hbs]: Go for it

File: README.md (hunk 2 above)
Comment (gcappello) [reply README.md]: done

File: app/views/partials/swagger/error-catalogue.hbs [hunk 3]
```diff
@@ -0,0 +1,56 @@
+{{!--
+  Original work Copyright (c) 2015 Nils Knappmeier
+  https://github.com/bootprint/bootprint-openapi
+
+  Renders the definition-section of the HTML-page.
+  @param {Definition[]} definitions a list of JSON-subschemas.
+  @api public
+--}}
+{{#if errorSection}}
```
Comment (jcerrada) [inline app/views/partials/swagger/error-catalogue.hbs:9]: Nice!

**Review by jcerrada — APPROVED** [review]

**Pull request merged**